composer.json

```diff
@@ -15,13 +15,13 @@
     "bin-dir": "bin"
   },
   "require": {
-    "php": ">=5.3.3",
+    "php": ">=5.5",
     "drush/drush": "dev-master#d1d13676f5beacaa9f8619088fe3ae45ea28a6cf",
-    "totten/php-symbol-diff": "dev-master#54f869ca68a3cd75f3386f8490870369733d2c23",
+    "totten/php-symbol-diff": "dev-master",
     "civicrm/upgrade-test": "dev-master#b93207fd28ac90426e9c95d572068f2e907b08e2",
     "drupal/coder": "dev-8.x-2.x-civi#a801890a82d52e23a83c5d820270b6e228843d79",
     "brianium/paratest": "dev-batching as 0.7",
-    "phpunit/phpunit": "3.7.27",
+    "phpunit/phpunit": ">=4",
```

Comment:
I want to clarify a little bit about what this means, because it's probably confusing.

Buildkit includes two copies of `phpunit` -- one via `composer.json` (v3) and one via PHAR (v4).  This patch changes the `composer.json` copy, which isn't actually used by any of our CI today. It may be used locally by some developers (because the name is more familiar), but I have no way to gauge this.

My gut reaction is that we should (1) remove phpunit from buildkit's `composer.json` and (2) optionally add a stub script `bin/phpunit` which calls `bin/phpunit4` or `bin/phpunit5` .

There is a longer discussion to be had about the general framing of per-project tooling vs global tooling (and how neither is really prefect for our ecosystem). But in terms of keeping changes small+manageable, I'd vote to remove `phpunit` from this `composer.json`.

Reply:
I would be ok with that as well

Reply:
I'm happy with that - anything that gets rid of php 3.7!

I still don't understand why we have this issue per se - is it just a vestige of prior support for php 5.4 (as I originally assumed) or is there some remaining complexity in our CI env?

Reply:
OK, closing this. Replaced by #408.